ruby/app/controllers/articles.rb

```diff
@@ -1,43 +1,37 @@
 class ArticleController
   def create_article(article)
-    article_exists = Article.where(tilte: 'title')
+    article_not_exists = ! (Article.where(:title => article['title']).empty?)
```

Comment:
If this looks confusing, it's intentional. They just need to sort out the correct logic in these statements.